package.tf

```diff
@@ -14,20 +14,32 @@ data "external" "archive_prepare" {
     })
 
     docker = var.build_in_docker ? jsonencode({
+      docker_pip_cache = var.docker_pip_cache
       docker_file    = var.docker_file
       docker_image   = var.docker_image
       with_ssh_agent = var.docker_with_ssh_agent
     }) : null
 
     artifacts_dir    = var.artifacts_dir
     runtime          = var.runtime
-    function_name    = var.function_name
     source_path      = var.source_path
     hash_extra       = var.hash_extra
-    hash_extra_paths = jsonencode(["${path.module}/lambda.py"])
+    hash_extra_paths = jsonencode(["${path.module}/package.py"])
   }
 }
 
+# This transitive resource used as a bridge between a state stored
+# in a Terraform plan and a call of a build command on the apply stage
+# to transfer a noticeable amount of data
+resource "local_file" "archive_plan" {
+  count = var.create && var.create_package ? 1 : 0
```

Comment:
Please run `pre-commit run -a` to fix the formatting of all files.

Reply:
Done.